Sources/Core/State/PlayingState.swift

```diff
@@ -140,6 +146,25 @@ final class PlayingState: PlayerState {
             }
         }
     }
+
+    //swiftlint:disable line_length
```

Comment:
comment lines are not well formatted & return to line to avoid disable line_length swiftlint rule.

Reply:
Done